.changeset/tasty-buses-jog.md

```diff
@@ -10,8 +10,70 @@ This is a breaking change.
 
 There are two ways to update:
 
-1. Set `experimental.useLegacyManyRelationNames: true` at the root of your config. This will make Keystone revert to the old relation names. This feature will be removed in a future release.
-2. Or, rename your many relation tables using a migration (see below)
+### Set `db.relationName` on many to many relation
+
+To avoid doing a migration, you can set `db.relationName` to the previous relation name on one side of a many to many relationship.
+
+For example, given a schema like this:
+
+```ts
+Post: list({
+  fields: {
+    tags: relationship({ ref: 'Tag.posts', many: true }),
+  },
+}),
+Tag: list({
+  fields: {
+    posts: relationship({ ref: 'Post.tags', many: true }),
+  },
+}),
+```
+
+Before this release, the generated Prisma schema looked like this:
+
+```prisma
+// This file is automatically generated by Keystone, do not modify it manually.
+// Modify your Keystone config when you want to change this.
+
+datasource postgresql {
+  url      = env("DATABASE_URL")
+  provider = "postgresql"
+}
+
+generator client {
+  provider   = "prisma-client-js"
+  output     = "node_modules/.prisma/client"
+  engineType = "binary"
+}
+
+model Post {
+  id   String @id @default(cuid())
+  tags Tag[]  @relation("Post_tags_Tag_posts")
+}
+
+model Tag {
+  id    String @id @default(cuid())
+  posts Post[] @relation("Post_tags_Tag_posts")
+}
+```
+
+You can add `db: { relationName: 'Post_tags_Tag_posts' }` to one side of the many to many relationship. It doesn't matter which side of the relationship it's on but it must only be on one side. 
```

Comment:
```suggestion
By adding `db: { relationName: 'Post_tags_Tag_posts' }` to one side of the many-to-many relationship; you can preclude yourself from a migration.

**Note:** It doesn't matter which side of the relationship you put this property,  but it should be only on one side; otherwise you will receive an error.
```